app/src/main/java/com/internship/move/ui/authentication/register/RegisterFragment.kt

```diff
@@ -28,6 +25,16 @@ class RegisterFragment : Fragment(R.layout.fragment_register) {
 
         initClickableText()
         initListeners()
+        initObservers()
+    }
+
+    private fun initObservers() {
+        viewModel.isLoading.observe(viewLifecycleOwner) { isLoading ->
+            binding.getStartedBtn.setIsLoading(isLoading)
+        }
+        viewModel.errorMessage.observe(viewLifecycleOwner) { message ->
```

Comment:
Personally I prefer leaving an empty line between observer setters, makes the code easier to read imo